config/tailwind.config.js

```diff
@@ -9,10 +9,16 @@ module.exports = {
   ],
   theme: {
     extend: {
+      colors:{
+        primary:"rgb(29 32 34)",
+        secondary: "#eeb14a",
+        hoverNavLink: "rgb(165 68 29)",
+        inputLabel: "rgb(238 177 74 / 73%)"
```

Comment:
Not sure how I feel about these class names. Everything else I see is using `kebab-case` but these are in `camelCase`